<?xml version="1.0" ?>
<sdf version="1.9">
  <model name="city_1">
    <static>true</static>
    <link name="link">
      <collision name="collision">
        <geometry>
          <mesh>
            <uri>model://city_1/meshes/city_1.stl</uri>
          </mesh>
        </geometry>
      </collision>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://city_1/meshes/city_1.stl</uri>
          </mesh>
        </geometry>
        <material>
            <ambient>0.8 0.8 0.8 1</ambient>
            <diffuse>0.8 0.8 0.8 1</diffuse>
            <specular>0.8 0.8 0.8 1</specular>
        </material>
      </visual>
    </link>
  </model>
</sdf>
